src/search/pruning/stubborn_sets_action_centric.cc

```diff
@@ -0,0 +1,62 @@
+#include "stubborn_sets_action_centric.h"
+
+using namespace std;
+
+namespace stubborn_sets {
+StubbornSetsActionCentric::StubbornSetsActionCentric(const options::Options &opts)
+    : StubbornSets(opts) {
+}
+// Relies on both fact sets being sorted by variable.
+static bool contain_conflicting_fact(const vector<FactPair> &facts1,
```

Comment:
I would move this function out of the way of the all of the class's definitions, i.e., above the constructor.